docs/config.json

```diff
@@ -1264,10 +1264,6 @@
             {
               "title": "PostgreSQL",
               "slug": "/database-access/auto-user-provisioning/postgres/"
-            },
-            {
-              "title": "Database Access Controls",
-              "slug": "/database-access/auto-user-provisioning/database-access-controls/"
```

Comment:
Should we add a redirect?

Reply:
We're going to be overhauling the organization of the docs site this/next month, so I am holding off on introducing new redirects. We are close to our redirect limit, and are considering new approaches for handling these.